Fixing formatting in EBS API Section (#134)
C2Devel/terraform-provider-croccloud: docs/resources/volume_attachment.md

Reviewer: ```suggestion
* `generated_device_name` - The device name generated during attaching. The value can be changed.
```

Reviewer: ```suggestion
* `instance_id` - The ID of the instance.
```

Reviewer: ```suggestion
* `volume_id` - The ID of the volume.
```

>   Manages an EBS volume attachment

Reviewer 2: ```suggestion
  Manages a volume attachment.
```

}
```

## Argument reference

The following arguments are required:

* `instance_id` - (Required, Forces new resource, String) The ID of the instance to attach to.
* `volume_id` - (Required, Forces new resource, String) The ID of the volume to be attached.

The following arguments are optional:

* `device_name` - (Optional, Editable, String) The device name to expose to the instance.

    ~> **Note** This argument is deprecated. Its value is ignored. The device name will be generated during attaching and can be changed.

* `skip_destroy` - (Optional, Editable, Boolean) Indicates whether to detach the volume on resource destroy or skip the detachment, but still remove the volume's attachment from the Terraform state.
    * _Default value:_ `false`

>       ~> **Note**  This can be useful if the volume was created outside of Terraform and you want to keep it attached to the instance.

Reviewer 2: ```suggestion
    ~> **Note**  This can be useful if the volume was created outside of Terraform and you want to keep it attached to the instance.
```

}
```

## Argument reference

The following arguments are required:

* `instance_id` - (Required, Forces new resource, String) The ID of the instance to attach to.
* `volume_id` - (Required, Forces new resource, String) The ID of the volume to be attached.

The following arguments are optional:

* `device_name` - (Optional, Editable, String) The device name to expose to the instance.

    ~> **Note** This argument is deprecated. Its value is ignored. The device name will be generated during attaching and can be changed.

* `skip_destroy` - (Optional, Editable, Boolean) Indicates whether to detach the volume on resource destroy or skip the detachment, but still remove the volume's attachment from the Terraform state.
    * _Default value:_ `false`

>    ~> **Note**  This can be useful if the volume was created outside of Terraform and you want to keep it attached to the instance.

Reviewer 2: ```suggestion
     ~> **Note**  This can be useful if the volume was created outside of Terraform and you want to keep it attached to the instance.
```

* `stop_instance_before_detaching` - (Optional, Editable, Boolean) Indicates whether the instance should be stopped before detaching the volume.
    * _Default value:_ `false`

## Attribute reference

### Supported attributes

In addition to all arguments above, the following attributes are exported:

* `generated_device_name` - (String) The device name generated during attaching. The value can be changed.
* `instance_id` - (String) The ID of the instance.

> * `volume_id` - (String) The ID of the volume.

Reviewer 2: Уже есть в аргументах

}
```

## Argument reference

The following arguments are required:

* `instance_id` - (Required, Forces new resource, String) The ID of the instance to attach to.
* `volume_id` - (Required, Forces new resource, String) The ID of the volume to be attached.

The following arguments are optional:

* `device_name` - (Optional, String) The device name to expose to the instance.

    ~> **Note** This argument is deprecated. Its value is ignored. The device name will be generated during attaching and can be changed.

* `skip_destroy` - (Optional, Boolean) Indicates whether detach the volume or not before destroying the instance.
    * _Valid values:_ `false`, `true`

      ~> **Note on the `skip_destroy` attribute:** Set its value to `true` if you *don't* want Terraform to detach the volume when destroying the instance.
      Terraform will only remove the volume's attachment from its own records.

>       This is useful when the volume was created outside of Terraform and you want to keep it attached to the instance.

Reviewer 2: ```suggestion
* `skip_destroy` - (Optional, Boolean) Indicates whether to detach the volume on resource destroy or skip the detachment, but still remove the volume's attachment from the Terraform state.
    * _Default value:_ `false`

      -> **Note**  This can be useful if the volume was created outside of Terraform and you want to keep it attached to the instance.
```

У параметра смысл немного другой.

* `stop_instance_before_detaching` - (Optional, Boolean) Indicates whether the instance should be stopped before detaching the volume.

>     * _Valid values:_ `false`, `true`

Reviewer 2: ```suggestion
    * _Default value:_ `false`
```

     ~> **Note on the `skip_destroy` attribute:** Set its value to `true` to ensure that the target instance is stopped before trying to detach the volume.

>         If the instance is not stopped already, then Terraform will stop it. 

Reviewer 2: Можно целиком удалить note, дублирует описание аргумента

The following arguments are supported:

> * `instance_id` - (Required) ID of the instance to attach to.

Reviewer: ```suggestion
* `instance_id` - (Required) The ID of the instance to attach to.
```

> * `volume_id` - (Required) ID of the volume to be attached.

Reviewer: ```suggestion
* `volume_id` - (Required) The ID of the volume to be attached.
```

* `device_name` - (Optional) The device name to expose to the instance.

> ~> **Note** The parameter `device_name` is deprecated. Its value is ignored.

Reviewer: Note относится к конкретному аргументу, так что лучше вставить его внутрь пункта (сделать нужный отступ)
```suggestion
  ~> **Note** The parameter `device_name` is deprecated. Its value is ignored.
```

Reviewer: должно выглядеть как-то так:
<img width="732" height="192" alt="image" src="https://github.com/user-attachments/assets/3f12181d-8119-440e-8d41-81d84d5b9fbc" />

> ~ > **Note** The parameter `device_name` is deprecated. Its value is ignored.

Reviewer: Note сделать с отступом, чтобы оно относилось к данному пункту, а не разбивало список

The device name will be generated during attaching and can be changed.

* `skip_destroy` - (Optional) Set this to `true` if you do not wish to detach the volume from the instance
  to which it is attached at destroy time, and instead just remove the attachment from Terraform state.

>   This is useful when destroying an instance which has volumes created by some other means attached.

Reviewer: тут чет путано написано, вероятно д.б.
```suggestion
  This is useful when destroying an instance which has attached volumes created by some other means.
```

The following arguments are supported:

* `instance_id` - (Required) The ID of the instance to attach to.
* `volume_id` - (Required) The ID of the volume to be attached.
* `device_name` - (Optional) The device name to expose to the instance.

    ~> **Note** The parameter `device_name` is deprecated. Its value is ignored.

> The device name will be generated during attaching and can be changed.

Reviewer 2: Не уверена, что стоит делать блоки с предупреждениями как бы подблоками для аргумента. Из-за того, что он рендерится в большой цветной блок дополнительное смещение слева, выглядит немного странно. 

Если предупреждение разбито на несколько строк (как собственно тут получилось), все строки кроме первой будут уже без отступа.

Author: Сбилдил, посмотрел, вроде адекватно) складывается корректное понимание, что блок о депрекейте относится именно к конкретному аргументу.
При рендере разбивки на строки нет.
<img width="1063" height="259" alt="Снимок экрана 2026-03-24 151446" src="https://github.com/user-attachments/assets/eb7a6fdc-b78c-47a9-88e7-db0552155120" />

Author: @PixarV

Reviewer 2: Я имела ввиду вид до рендера. Если вам больше нравится сдвинутый блок, окей

//блок сдвинут, потому что он относится к конкретному аргументу

In addition to all arguments above, the following attributes are exported:

* `generated_device_name` - The device name generated during attaching. The value can be changed.
* `instance_id` - The ID of the instance.
* `volume_id` - The ID of the volume.

### Unsupported attributes

~> **Note** This attribute may be present in the `terraform.tfstate` file, but it has a preset value and cannot be specified in configuration files.

The following attribute is not currently supported: `force_detach`.

> ## Timeouts

Reviewer: А это точно здесь д.б.? Уточнить у Риты

Author: Да, должен быть, уточнял ранее

> The `timeouts` block allows you to specify [timeouts] for certain actions:

Reviewer 2: Нужно докинуть метку со ссылкой в начало файла
